python/mxnet/gluon/contrib/nn/basic_layers.py

```diff
@@ -165,7 +165,10 @@ class SyncBatchNorm(BatchNorm):
 
     Standard BN [1]_ implementation only normalize the data within each device.
     SyncBN normalizes the input within the whole mini-batch.
-    We follow the sync-onece implmentation described in the paper [2]_.
+    We follow the implementation described in the paper [2]_.
+
+    Note: Current implementation of SyncBN does not support FP16 training.
```

Comment:
So does it not support training or inference?
You say training in this line, and refer to inference in the next line.

Reply:
SyncBN does not have FP16 support for both training and inference. But for FP16 inference, SyncBN can be replaced with nn.BatchNorm as they have similar functionality.